simul.spec

```
@@ -48,10 +47,10 @@ and coherence of parallel filesystems.
 %if %{with_mpich}
 %package mpich
 Summary: Simul for MPICH
-%if "%{_host_vendor}" == "suse"
-BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel glibc
+%if (0%{?suse_version} > 0)
```

Versions for distros should be as conservative as needed/known and only made more liberal as the need is discovered. 
 Since we don't know (or support even) versions less than 15.x:

```suggestion
%if (0%{?suse_version} >= 1500)
```